lib/disposable.dart

```diff
@@ -181,4 +193,10 @@ abstract class Disposable implements _Disposable {
     _didDispose.complete();
     return null;
   }
+
+  void _throwIfNull(dynamic subscription, String name) {
```

Comment:
If the actual solution was a bit more elegant I would definitely support pulling it into w_common proper. But as it stands, it's not. Hopefully non-nullable types will save us from this nightmare... https://github.com/dart-lang/sdk/issues/27231 :wink: